pyproject.toml

```diff
@@ -30,25 +30,26 @@ classifiers = [
     "Programming Language :: Python :: 3.13",
 ]
 dependencies = [
-    "Sphinx>=6.1.0",
-    "pydata-sphinx-theme>=0.15.4,<0.17",
-    "Jinja2>=3.1.2",
+    "beautifulsoup4==4.13.4",
```

Comment:
Not a big fan of adding this dependency just for adding a "Home" directory knowing that a simple would do the job...

```rst
Home <self>
```